evaluation/evaluate_metrics/base_metric.py

```diff
@@ -11,9 +11,10 @@
 class BaseMetric(ABC):
     METRIC_NAME = "name_of_metric"
 
-    DISPLAY_NAME = "Base Evaluation Metric Rating"
-    SHORT_NAME = "Base Metric"
-    NOTE = "Rating Score"
+    # Constants for diagram generation
+    DISPLAY_NAME = "Title of Metric Diagram"
+    SHORT_NAME = "Short Name"
+    NOTE = "Y Axis Label"
```

Comment:
It seems like a couple of changes have kept some code from an earlier version of the evaluation-graph branch which are outdated. Merge with the main branch again could be a solution to that but that might bring more conflicts,  I'll point them out and you can manually change them by referencing the current main branch.